testsuite/testcases/src/test/cases/over-2gb-input/.gitignore

```diff
@@ -0,0 +1 @@
+Over2GbInput.java
```

Comment:
Generated files shouldn't be in `src/test`

Reply:
This file is the over-2GB input file generator; it is not itself generated.